src/properties.cpp

```diff
@@ -933,71 +933,1521 @@ namespace Exiv2 {
-        { "AOSourceInvNo",           N_("Artwork or object-Source inventory number"), "Text",             xmpText, xmpExternal, N_("The inventory number issued by the organisation or body holding and registering the artwork or object in the image.") },
-        { "AOTitle",                 N_("Artwork or object-Title"),         "Lang Alt",                   langAlt, xmpExternal, N_("A reference for the artwork or object in the image.") },
+        { "AOCopyrightNotice",       N_("Artwork or object-Copyright notice (deprecated)"), "Text",       xmpText, xmpExternal, N_("Deprecated, use the AOCopyrightNotice tag in ArtworkOrObjectDetails struct instead. Contains any necessary copyright notice for claiming the intellectual property for artwork or an object in the image and should identify the current owner of the copyright of this work with associated intellectual property rights.") },
+        { "AOCreator",               N_("Artwork or object-Creator (deprecated)"), "seq ProperName",      xmpSeq,  xmpExternal, N_("Deprecated, use the AOCreator tag in ArtworkOrObjectDetails struct instead. Contains the name of the artist who has created artwork or an object in the image. In cases where the artist could or should not be identified the name of a company or organisation may be appropriate.") },
+        { "AODateCreated",           N_("Artwork or object-Date Created (deprecated)"), "Date",           xmpText, xmpExternal, N_("Deprecated, use the AODateCreated tag in ArtworkOrObjectDetails struct instead. Designates the date and optionally the time the artwork or object in the image was created. This relates to artwork or objects with associated intellectual property rights.") },
+        { "AOSource",                N_("Artwork or object-Source (deprecated)"),       "Text",           xmpText, xmpExternal, N_("Deprecated, use the AOSource tag in ArtworkOrObjectDetails struct instead. The organisation or body holding and registering the artwork or object in the image for inventory purposes.") },
+        { "AOSourceInvNo",           N_("Artwork or object-Source inventory number (deprecated)"), "Text",xmpText, xmpExternal, N_("Deprecated, use the AOSourceInvNo tag in ArtworkOrObjectDetails struct instead. The inventory number issued by the organisation or body holding and registering the artwork or object in the image.") },
+        { "AOTitle",                 N_("Artwork or object-Title (deprecated)"),   "Lang Alt",            langAlt, xmpExternal, N_("Deprecated, use the AOTitle tag in ArtworkOrObjectDetails struct instead. A reference for the artwork or object in the image.") },
         // End of list marker
         { nullptr, nullptr, nullptr, invalidTypeId, xmpInternal, nullptr }
     };
 
+    //! XMP Iptc4xmpCore:Scene
```

Comment:
Where are these tags coming from? I could not find them in the new standard. 

I think it would make easier for the people reading the code to add some comments indicating public resources / standard from where certain tags have been added.

Reply:
The tags come from http://cv.iptc.org/newscodes/scene/ . I have added a comment explaining this.